Library/ViewModels/DiscoveryPostcardViewModel.swift

```diff
@@ -197,9 +203,17 @@ public final class DiscoveryPostcardViewModel: DiscoveryPostcardViewModelType,
       .map { $0.personalization.friends == nil || $0.personalization.friends?.count ?? 0 == 0 }
       .skipRepeats()
 
+    self.fundingProgressContainerViewHidden = project
+      .map { $0.state == .canceled || $0.state == .suspended }
+
+    self.fundingProgressBarViewHidden = project
+      .map { $0.state == .failed }
+
     // a11y
     self.cellAccessibilityLabel = project.map(Project.lens.name.view)
-    self.cellAccessibilityValue = project.map(Project.lens.blurb.view)
+
+    self.cellAccessibilityValue = Signal.combineLatest(project, self.projectStateTitleLabelText)
+      .map { project, projectState in "\(project.blurb)\(projectState)" }
```

Comment:
this interpolation might be weird if the blurb doesn't have a space/punctuation at the end, e.g. "This is some project blurbSuccessfully funded". instead we could interpolate it as `"\(project.blurb). \(projectState)"`.

also, `combineLatest` will cause some over emissions cause it will emit once for each of the signals, hence twice overall. a better operator would be `zip` which requires each signal to emit once before emitting again.